README.md

```diff
@@ -11,12 +11,13 @@ The .NET [OpenTelemetry](https://opentelemetry.io/) client.
 
 ## Installation
 
-This repository includes multiple installable packages. The `OpenTelemetry.Api`
-package includes abstract classes and no-op implementations for the
-[OpenTelemetry API
-specification](https://github.com/open-telemetry/opentelemetry-specification).
+This repository includes multiple installable packages, available on
+[NuGet](https://www.nuget.org/profiles/OpenTelemetry):
 
-The `OpenTelemetry` package is the reference implementation of the API.
+* The `OpenTelemetry.Api` package includes abstract classes and no-op

```

Comment:
Since the SHIM is also part of this package, should we remove the term "abstract classes"

Reply:
Let me rephrase this.

Reply:
Updated the wording.